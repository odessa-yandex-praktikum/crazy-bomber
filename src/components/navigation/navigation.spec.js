import React from 'react';
import renderer from 'react-test-renderer';
import {Navigation} from './index';

jest.mock('react-router-dom', () => {
    return {
        Link: () => <div>this is mock</div>,
    };
});

it('renders correctly', () => {
    const navigationItems = ['start'];
    const tree = renderer.create(<Navigation navigationItems={navigationItems} />).toJSON();
    expect(tree).toMatchSnapshot();
});
