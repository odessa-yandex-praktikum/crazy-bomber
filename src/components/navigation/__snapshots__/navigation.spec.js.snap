// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="navi__list"
>
  <li
    className="navi__item"
  >
    <div>
      this is mock
    </div>
  </li>
</ul>
`;
